components/com_content/src/Controller/ArticleController.php

```diff
@@ -403,6 +404,67 @@ public function save($key = null, $urlVar = 'a_id')
 		return $result;
 	}
 
+	/**
+	 * Method to save article's title
+	 *
+	 * @return void
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	public function FEInlineEdition()
+	{
+		$this->checkToken();
+
+		$itemprop = $this->input->getString('itemprop');
+		$articleId = $this->input->getInt('a_id');
+
+		$newValue = '';
+
+		if ($itemprop == 'articleBody')
+		{
+			$newValue = $this->input->getRaw('value');
+		}
+		else
+		{
+			$newValue = $this->input->getString('value');
+		}
+
+		if (!$itemprop || !$articleId || !$newValue || !$this->allowEdit(['id' => $articleId]))
+		{
+			echo new JsonResponse(['saved' => false]);
+		}
+		else
+		{
+			$field = '';
+
+			if ($itemprop == 'headline')
```

Comment:
Add strict comparison.

See if you can rewrite the triple if/elseif/else to a simpler statement so it becomes an early return. Just like we removed the else statement above.